tools/BUILD

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+package(default_visibility = ["//visibility:public"])
```

Comment:
can you add a module docstring (e.g., https://github.com/bazelbuild/rules_docker/commit/8010a50ef03d1e13f1bebabfc625478da075fa60)

Reply:
Do we want to do this in the BUILD file?

Reply:
oh sorry, did not see it down there and commented here by accident